Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.6
+RUN apk --no-cache add nodejs-current nodejs-npm
+RUN npm set progress=false && npm install -g nba-go
+CMD ["nba-go", "game", "-t"]
```

Comment:
As i'm not that familiar with docker, does this mean the docker container will always only run the command `nba-go game -t` ?

Reply:
it means when the docker starts it will execute that line

Reply:
by default with no arguments that command will be run. To run other commands, you can override the default at run time. For example:

```
$ docker run -it nba-go:latest nba-go player Ginobili -i #GoSpursGo
```

Reply:
Hi @hhoover ,
it loos great! Sorry for late message, cuz I just forgot to ask you why it should execute `nba-go game -t` when the docker container started.
Could you add how to override the default CMD in README? Thanks!

Reply:
done!